packages/js/src/settings/app.js

```diff
@@ -70,8 +71,8 @@ const Menu = ( { postTypes, taxonomies, idSuffix = "" } ) => {
 
 	return <>
 		<header className="yst-px-3 yst-mb-6 yst-space-y-6">
-			<Link to="/" className="yst-inline-block">
-				<YoastLogo className="yst-w-40" { ...svgAriaProps } />
+			<Link to="/" className="yst-inline-block yst-rounded-md focus:yst-border focus:yst-outline-none focus:yst-ring-2 focus:yst-ring-offset-2 focus:yst-ring-primary-500"  tabindex={ 0 }>
+				<YoastLogo className="yst-w-40" { ...svgAriaProps } ariaLabel={ `Yoast SEO ${isPremium ? "Premium " : "" }settings` } />
```

Comment:
The logo does not represent the settings. I guess it is more about the link pointing to the settings again? 🤔 
Either way, I think you should translate the `settings` part here if you add it.

Reply:
But it is said in the issue not to translate this label for some reason.